Components/firebase-messaging-42.1001.0.0/samples/FirebaseMessagingQuickstart/FirebaseMessagingQuickstart/Resources/AboutResources.txt

```diff
@@ -0,0 +1,44 @@
+Images, layout descriptions, binary blobs and string dictionaries can be included 
```

Comment:
These don't belong in our project, should gitignore it because it causes necessary commits. I made issue #58 for this